tests/game/test_buildings.py

```diff
@@ -173,21 +174,58 @@ def test_build_tear(s, p):
 
 
 @game_test(timeout=60)
-def test_tree_production(s, p):
-	"""Check whether trees produce wood"""
+def test_tree_wood_production(s, p):
+	"""
+	Check whether trees produce wood
+	"""
 	settlement, island = settle(s)
+
+	# create building tree at coordinates (30,35)
 	tree = Build(BUILDINGS.TREE, 30, 35, island, settlement=settlement)(p)
 
+	# number of wood units to produce
 	n = 20
 
+	# create inventory for building tree
```

Comment:
This doesn't actually create anything, it just gets a reference to the inventory. I'd drop the comment.